coralproject/talk
# [CORL-2564] Use review counts for R&R stories in count scripts

## What does this PR do?

Updates the count scripts to show ratings for Ratings & Reviews stories instead of comment counts.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No new feature flags.

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes added.

## How do I test this PR?

- Create a R&R story
- Ensure you embed the count script on the story page and also on another page to reference the story like a main page on a site with links to many stories where you'd want to see comment counts beside the story link
- make some reviews
- See that the count scripts show `Ratings` / `Rating` in their counts
 
## How do we deploy this PR?

No special considerations.
